src/server/game/Entities/Player/Player.h

```diff
@@ -2182,6 +2183,10 @@ class TC_GAME_API Player final : public Unit, public GridObject<Player>
 
         void RemovedInsignia(Player* looterPlr);
 
+        void InitAdvancedFly();
```

Comment:
InitAdvancedFlight instead?

Reply:
Hum, i can switch to "InitAdvancedFlying" as this is the opcodes names

Reply:
Following opcode naming sounds good to me

Reply:
AdvFlying it is then, it will also match the movementInfo member

Reply:
sounds better to me too